### PR TITLE
Configuration for custom-messaging-in-aboutwelcome-for-chrome-users-to-import

### PR DESCRIPTION
**Pull request by tdsmith**

### CODE REVIEW

**Comment by tdsmith** [comment]
@SuYoungHong Figured I'd start working on this while I was thinking about it -- here's a fairly compact way to count users that have performed an import from Chrome.

**Comment by SuYoungHong** [comment]
very nice. quick question, is there any specific reason are we limiting to chrome imports specifically (versus any non-firefox browser)?

**Comment by tdsmith** [comment]
I thought it seemed like the natural thing to measure for Chrome-targeted messaging but I see that's not what you specced; thanks for calling that out.

**Comment by SuYoungHong** [comment]
I think checking both is good. I've found that measuring the general case is important context even when targeting specific cases.

File: custom-messaging-in-aboutwelcome-for-chrome-users-to-import.toml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    "exposed_treatment_a",
+    "exposed_treatment_b",
+    "exposed_control",
+]
+
+# Key outcomes
+
+[metrics.opened_migration_wizard]
+select_expression = """COALESCE(SUM(
+    (SELECT SUM(value) FROM UNNEST(mozfun.hist.extract(payload.histograms.fx_migration_entry_point).values))
```
Comment (tdsmith) [inline custom-messaging-in-aboutwelcome-for-chrome-users-to-import.toml:33]: this is "count of the number of observations in the histogram" and I should probably write a helper for it

**Comment by tdsmith** [comment]
This covers the analysis questions on https://mana.mozilla.org/wiki/display/FIREFOX/Experiment%3A+Custom+Messaging+in+about%3Awelcome+for+Chrome+Users+to+Import; would you expect more detail about the other exits from the funnel?

**Pull request merged**